### ISSUE TITLE
Can't download / load images after uploading them

### ISSUE THREAD

**Issue by GK-GreyGhost** [pocketbase/pocketbase#45]
I can't download / load images after uploading them
```
const img = new Image();
img.src = `/api/files/${item['@collectionId']}/${item.id}/${item.avatar}`;
document.body.appendChild(img);
//returns net::ERR_CONTENT_LENGTH_MISMATCH 400 (Bad Request)
```

when trying to download through the admin panel it also fails with the error "Couldn't download - something went wrong"
though the file does exist and I can open it in pb_data/storage

tried in both Edge and Firefox on windows 10 (all up to date)
no adblockers running

How should I proceed?

Thanks,
Harrison

**Comment by ganigeorgiev** [comment]
> when trying to download through the admin panel it also fails with the error "Couldn't download - something went wrong"

Assuming that your `item.avatar` is image filename, does this means that you can't see its preview/thumb in the admin UI?
If you are not seeing the preview/thumb in the admin, could you post the response from the browser "Inspect tool > Network" tab?

**Comment by GK-GreyGhost** [comment]
item.avatar is the image filename

I cannot see it in the admin UI
![image](https://user-images.githubusercontent.com/1384496/178152299-fc4e9c47-cac2-4aab-b5f4-ee6ff20fb288.png)

"Inspect tool > Network > Response > This request has no response data available."

![image](https://user-images.githubusercontent.com/1384496/178152644-5a08e5f1-226c-4a7e-9e6e-dc5980622bca.png)

**Comment by ganigeorgiev** [comment]
This is strange. I think the only place in the file api that could return 400 is when we initialize the file storage system - https://github.com/pocketbase/pocketbase/blob/master/apis/file.go#L53.

But if you were able to upload it, then you should be able also to read it.

I'll try to test it in my Windows 10 VM, maybe is a platform specific issue related to the mime types similar to issue #6.

**Comment by GK-GreyGhost** [comment]
I do need to run it with "--http="127.0.0.1:8090"" otherwise it exits with "2022/07/10 09:33:53 listen tcp: lookup localhost on 192.168.0.1:53: no such host"

I just checked and the thumbnail is also generated correctly, I can open it and it displays in my file browser

**Comment by ganigeorgiev** [comment]
I've just tested it and indeed there is an issue when using `time.LoadLocation` on Windows - https://github.com/golang/go/issues/38453.

 `time.LoadLocation` is currently used to set the `Last-Modified` header timestamp when serving files.
I'll do some tests and will see if I can manage to fix it without loading tzdata for the OS.

**Comment by ganigeorgiev** [comment]
This should be fixed in the latest [v0.2.2 release](https://github.com/pocketbase/pocketbase/releases).

_To avoid embedding the [tzdata](https://pkg.go.dev/time/tzdata) package, which will increase the binary size with almost 0.5MB, the current implementation will fallback to default browser value and will skip the explicit `Last-Modified` header if fails to `time.LoadLocation` fails._

**Comment by GK-GreyGhost** [comment]
it works, thank you! 😁
